src/pages/world.tsx

```diff
@@ -151,8 +151,9 @@ export default function WorldPage() {
       <Navigation />
       <Container style={{ paddingTop: "75px" }}>
         <ItemLayout>
-          <WorldDescription
```

Comment:
We are expecting that modification over the definitions and info of a world, that is why it was on a different component, please revert this change now so we don't have to do it later